tests/cli/fixtures/cli_kg.py

```diff
@@ -54,6 +54,7 @@ def callback(request):
             status_code = 404 if authorization == "Bearer renku-token" else 401
             return status_code, {"Content-Type": "application/json"}, ""
 
+        response.add_passthru("https://pypi.org/")
```

Comment:
Why is this needed?

Reply:
I got an error running tests, it hit the code that checks if an update is available for renku, but then responses complained that there was no hook defined for pypi.org url that gets used in the version check.